test/integ/cli/test_utils.py

```diff
@@ -0,0 +1,64 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+
+from dataclasses import dataclass
+from pathlib import Path
+import boto3
+from pytest import TempPathFactory
+import os
+
+from deadline.job_attachments._aws.deadline import get_queue
+from deadline.job_attachments.asset_manifests.hash_algorithms import HashAlgorithm, hash_file
+from deadline.job_attachments.asset_manifests.versions import ManifestVersion
+from deadline.job_attachments.models import Attachments, JobAttachmentS3Settings
+from deadline_test_fixtures.deadline import DeadlineClient
+
+
+class JobAttachmentTest:
```

Comment:
All re-factored code from the general integ test. This has no dependencies to the existing tests and it will allow us a clean cut when we delete the old `sync_input` and `sync_output` code.